engine/crates/parser-openapi/src/tests/snapshots/parser_openapi__tests__flat_output.snap

```diff
@@ -1,13 +1,13 @@
 ---
-source: common/parser-openapi/src/lib.rs
+source: engine/crates/parser-openapi/src/tests/mod.rs
 expression: registry.export_sdl(false)
 ---
 type Mutation {
 	updatePet(input: PetstorePetInput!): PetstorePet
 	addPet(input: PetstorePetInput!): PetstorePet
 	uploadFile(petId: Int!, additionalMetadata: String): PetstoreApiResponse
-	placeOrder(input: PetstoreOrderInput!): PetstoreOrder
-	createUsersWithListInput(input: [PetstoreUserInput!]!): PetstoreUser
+	placeOrder(input: PetstoreOrderInput): PetstoreOrder
+	createUsersWithListInput(input: [PetstoreUserInput!]): PetstoreUser
```

Comment:
I was puzzled by these two lines but I think the change is correct. Just like for fields, we used to add a `NonNull` wrapper for request bodies if the referenced type was not declared as nullable. But the source of truth for the optionality of the body is the `required` key in `requestBody` in the spec, defaulting to `false` (source for that last point: https://swagger.io/docs/specification/describing-request-body/).

So I think this is correct.

Reply:
Hmm, yeah.  Doesn't make sense as an API, but it does seem to be correct.